crashtracker/src/crash_handler.rs

```diff
@@ -244,6 +284,13 @@ fn emit_siginfo(w: &mut impl Write, signum: i32) -> anyhow::Result<()> {
 }
 
 fn handle_posix_signal_impl(signum: i32) -> anyhow::Result<()> {
+    static NUM_TIMES_CALLED: AtomicU64 = AtomicU64::new(0);
+    if NUM_TIMES_CALLED.fetch_add(1, SeqCst) > 0 {
+        // In the case where some lower-level signal handler recovered the error
+        // we don't want to spam the system with calls.  Make this one shot.
+        return Ok(());
+    }
+
     // Leak receiver, config, and metadata to avoid calling 'drop' during a crash
     let receiver = RECEIVER.swap(ptr::null_mut(), SeqCst);
     anyhow::ensure!(!receiver.is_null(), "No crashtracking receiver");
```

Comment:
I guess this is somewhat harmless, but wasn't the existing code already handling this correctly? E.g. the first crash would grab the receiver and any other crash would return with the "No crashtracking receiver" option.